app/src/main/java/com/example/geoquiz/QuizActivity.kt

```diff
@@ -150,10 +154,17 @@ class QuizActivity : AppCompatActivity() {
             ).show()
         }
 
-        cheatButton.setOnClickListener {
+        cheatButton.setOnClickListener { view ->
             val answerIsTrue = quizViewModel.currentQuestionAnswer
             val intent = CheatActivity.newIntent(this@QuizActivity, answerIsTrue)
```

Comment:
When things get crowded inside lambda brackets, that's a sign that we can move code to a function that will be called upon button press. I would move the code to a `openCheatActivity()` function